dll/win32/comdlg32/colordlg.c

```diff
@@ -1236,11 +1250,19 @@ static INT_PTR CALLBACK ColorDlgProc( HWND hDlg, UINT message,
 	                if (CC_WMMouseMove(lpp, lParam))
 			  return TRUE;
 			break;
+#ifdef __REACTOS__
```

Comment:
I think you could just add a `// ReactOS: The following comment doesn't apply` ,
instead of having a full-blown #ifdef guard here :thinking:

Reply:
Fixed.